packages/ui-app/src/TxButton.tsx

```diff
@@ -48,24 +96,20 @@ class TxButtonInner extends React.PureComponent<Props & InjectedProps> {
   }
 
   private send = (): void => {
-    const { accountId, api, onClick, onFailed, onSuccess, onUpdate, params = [], queueExtrinsic, tx } = this.props;
+    const { accountId, accountNonce, onClick, onFailed, onSuccess, onUpdate, queueExtrinsic, tx } = this.props;
+    const { extrinsic } = this.state;
 
     assert(tx, 'Expected tx param passed to TxButton');
 
-    const [section, method] = tx.split('.');
-
-    assert(api.tx[section] && api.tx[section][method], `Unable to find api.tx.${section}.${method}`);
-
-    const extrinsic: any = api.tx[section][method](
-      ...(isFunction(params) ? params() : params)
-    );
-
     queueExtrinsic({
       accountId,
       extrinsic,
       txFailedCb: onFailed,
       txSuccessCb: onSuccess,
-      txUpdateCb: onUpdate
+      txUpdateCb: onUpdate,
+      ...(accountNonce ?
```

Comment:
We could have left this our, only used in transfer. I'm guessing it is actually a hold-over from before the API automagically loaded the nonce.